web/src/components/tabs/Tabs/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { ReactElement, useState } from 'react'
+import { useTheme } from 'react-jss'
+
+import { Tab } from '../Tab'
+
+import useStyles from './styles'
+
+export type TabsProps = {
+  size?: 'large' | 'regular' | 'small'
+  align?: 'left' | 'right' | 'justify'
+  margin?: number
+  width?: number
```

Comment:
do we really want to have a width for a tabs component?
Shouldn't it be always just 100% and the parent limits the width?

If we keep it, it should also allow percentual width (allow string aswell) but I would rather remove it here.

Same for margin. We should think about having the same way to style elements (width, margin) on every component.

Reply:
E.g.: charka-ui has `Box` component that has some basic styling options
https://chakra-ui.com/box

Tab's inherits Box https://chakra-ui.com/tabs

Reply:
I described why I added `margin` and `width` here - https://github.com/ubergrape/grape-ds/pull/33#issuecomment-720353906

I removed them for now because in the reality of usage they really don't have any sense.